### PR TITLE
docs: change markup's title

### PR DESCRIPTION
**Pull request by XhmikosR**
Since we add a title in the iframe itself and we have an `aria-label` this was redundant.

@patrickhlauke can you confirm everything looks good now please? ❤

Preview: <https://optimistic-swanson-3a1e25.netlify.app/>

### CODE REVIEW

**Comment by patrickhlauke** [comment]
as discussed on twitter, yes...this seems to be the least bad way to work around it (balancing the need for iframes to provide a title, as otherwise accessibility checkers - usually rightly - complain, and the need for not being overly repetitive and verbose when navigating with a screen reader)

**Comment by patrickhlauke** [comment]
(and, again, just to reiterate that the `<title>` in the actual document loaded inside the `<iframe>` is generally not announced - at least not in from memory/from testing in a few browser/AT combos on regular settings, regardless ... the main issue here is more that the `title="..."` and then the `aria-label` on the actual links was a bit too much ... `<title>` itself is irrelevant)

**Comment by XhmikosR** [comment]
Thanks for chiming in!

Can you check please NDA or whatever program you use how is the situation for a given embedded button from the main page on the live docs and on this [PR preview](https://optimistic-swanson-3a1e25.netlify.app/)?

So that I have a clear view of the issue. Thanks!

**Comment by patrickhlauke** [comment]
the various browser/AT combos all behave subtly differently when encountering frames, and it also depends how you navigate (e.g. TABbing from control to control versus using reading keys etc). But yes, from quick testing, this is the least bad approach (fundamentally, because of the use of `<iframe>`, this will always be more verbose than strictly necessary, but assuming that pages will use these sparingly, it's not as bad).

as an example of convoluted/verbose behavior, on MacOS/Chrome when encountering an iframe, VoiceOver will have two separate stops when you navigate with reading keys (CTRL+ALT+cursor left/right): one for the "group", and then one for the "frame" (and then the user has to actively say they want to navigate into the iframe, and then they'll get to the link inside). For the frame, it seems Chrome does expose/use the `<title>` of the iframe itself (no other browser/AT combo I checked does this though, so uncommon).

The old/current iframe is thus experienced like this

Star twbs/bootstrap on GitHub, group
Star twbs/bootstrap on GitHub, frame

With this PR, it at least limits this a bit to

GitHub, group
Star twbs/bootstrap on GitHub, frame

MacOS/Safari is less verbose with VoiceOver, and only announces the frame itself (using the `title="..."`).

The old/current one is announced as

Star twbs/bootstrap on GitHub, frame

With this PR, it's

GitHub, frame

Windows/Chrome and NVDA only announces the frame (no extra "group"), so here the announcement is similar to MacOS/Safari. If navigating by TAB, it currently announces quite verbosely, when jumping to a link inside the iframe from outside the iframe:

Star twbs/bootstrap on GitHub  frame, Star twbs/bootstrap on GitHub  visited  link

With this PR, it limits it to something more manageable

GitHub  frame, Star twbs/bootstrap on GitHub  visited  link

Anyway, long story short, this change reduces unnecessary verbosity a bit. For the rest, you just gotta live with it as it's due to the nature of the solution that was chosen.

**Comment by XhmikosR** [comment]
Thanks @patrickhlauke!

@mdo LGTY? We should get this merged along with a couple of the other open PRs and make a new patch release soon.
